graders/c/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:24.04
-
 LABEL maintainer="jonatan@yorku.ca"
 
+ARG CACHEBUST=1
```

Comment:
Can we start these all off with a current-ish timestamp instead of `1`?

Reply:
Fixed in [5cd46e3](https://github.com/PrairieLearn/PrairieLearn/pull/11696/commits/5cd46e341504107d342b6f95ad33515b8fee0b50).